src/logic/quickstart.utils.ts

```diff
@@ -0,0 +1,63 @@
+import { SuperJson } from '@superfaceai/one-sdk';
+import { join as joinPath } from 'path';
+
+import { EXTENSIONS } from '..';
+import { execShell, exists } from '../common/io';
+import { LogCallback } from '../common/log';
+
+export async function installSdk(options?: {
+  logCb?: LogCallback;
+  warnCb?: LogCallback;
+}): Promise<void> {
+  const result = await execShell(
+    `${
+      (await exists('yarn.lock')) ? 'yarn add' : 'npm install'
```

Comment:
in the future, i'd abstract all package manager shenanigans to it's own class, which would detect (or even ask maybe) which pm to use and you'd just call `PackageManager.install('@superfaceai/one-sdk')` or something

Reply:
I would add this to backlog(beglog) 🤷